### PR TITLE
Add initial `ask` block to prompt for user input in Play Lab

### PR DESCRIPTION
**Pull request by joshlory**
This first iteration uses the browser's built-in `prompt()` API to block execution and get input from the user.

In the demo below, clicking the witch prompts for a new value.  Clicking the dog makes the actor say the current value.

ask-block
- [ ] Fix bug where setting the actor to a cat and a dog causes the prompt to appear twice.

### CODE REVIEW

File: apps/src/studio/blocks.js [hunk 1]
```diff
@@ -2419,6 +2419,42 @@ exports.install = function (blockly, blockInstallOptions) {
...
+   * Blocking prompt for user input.
+   */
+  blockly.Blocks.studio_ask = {
+    helpUrl: '',
+    init: function () {
+      var fieldLabel = new Blockly.FieldLabel(Blockly.Msg.VARIABLES_GET_ITEM);
+      // Must be marked EDITABLE so that cloned blocks share the same var name
+      fieldLabel.EDITABLE = true;
+      this.setHSV(312, 0.32, 0.62);
+      this.appendDummyInput().appendTitle('ask');
```
Comment (Hamms) [inline apps/src/studio/blocks.js]: should this (edit: and "to set" below) be internationalized?

File: apps/src/studio/blocks.js [hunk 2]
```diff
@@ -2419,6 +2419,42 @@ exports.install = function (blockly, blockInstallOptions) {
...
+      this.appendDummyInput()
+        .appendTitle(new Blockly.FieldImage(
+          Blockly.assetUrl('media/quote0.png'), 12, 12))
+        .appendTitle(new Blockly.FieldTextInput(''), 'TEXT')
+        .appendTitle(new Blockly.FieldImage(
+          Blockly.assetUrl('media/quote1.png'), 12, 12));
+      this.appendDummyInput().appendTitle('to set');
+      this.appendDummyInput()
+        .appendTitle(Blockly.Msg.VARIABLES_GET_TITLE)
+        .appendTitle(Blockly.disableVariableEditing ? fieldLabel
```
Comment (Hamms) [inline apps/src/studio/blocks.js:2444]: I'm not sure I understand what this is supposed to do

Comment (joshlory) [inline apps/src/studio/blocks.js:2444]: This is carried over from the `set_variable` block — it removes the dropdown button on levels that don't allow editing of variables (if the level builder has provided one variable they want the student to use).

**Comment by joshlory** [comment]
Confirmed the double-bind bug is a pre-existing issue.  I'm not going to block this change on fixing it.  Still need to remove the new block from the /p/playlab toolbox so it's not accessible to students yet.

File: apps/src/studio/blocks.js (hunk 1 above)
Comment (joshlory) [inline apps/src/studio/blocks.js]: Good catch, fixed in e72bf80b484ba159d321d44186717dbb5c96c23c.

**Comment by joshlory** [comment]
@Hamms Ready for another pass when you get a chance!

**Comment by Hamms** [comment]
LGTM!

**Pull request merged**